src/arduino/components/Pressure.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pressure.h"
+
+/**
+ * Returns pressure in PSI
+ *
+ * LOGIC:
+ * 1. Read sensor value
+ * 2. Convert sensor value to voltage
+ * 3. Convert voltage to pressure
+ * 4. Set pressure
+ * 5. Return pressure
+ */
+float Pressure::getPressure()
+{
+  double sensorRead = analogRead(sensorPin) * 0.0049;                                                                // Volts
+  double pressure = (((sensorRead - 0.1 * VOLTAGE_CONST) * (P_MAX - P_MIN)) / (0.8 * VOLTAGE_CONST)) + P_MIN + 0.36; // PSI
+  Pressure::pressure = pressure;
+  return pressure;
+}
+
+/**
+ * Returns true if pressure is within tolerance of the ideal pressure
+ */
+bool Pressure::PressureOk()
+{
+  return (pressure >= IDEAL_PRESSURE - PRESSURE_TOLERANCE && pressure <= IDEAL_PRESSURE + PRESSURE_TOLERANCE);
+}
+
+/**
+ * Turns compressor on or off
+ *
+ * LOGIC:
+ * if pressure is less than IDEA_PRESSURE or override is true:
+ *   turn on compressor
+ * else:
+ *   turn off compressor
+ */
+void Pressure::Pressurize(bool override)
```

Comment:
I'd love to see some clarity on when override gets used here

Reply:
override is used when the switch is pressed and cases like that

Reply:
You can also see it being used in the loop function